### PR TITLE
Move CI to supported queues

### PR DESCRIPTION
**Pull request by RussKeldorph**
Should also address #24568 for master

### CODE REVIEW

**Comment by RussKeldorph** [comment]
@MattGal @mthalman

File: eng/platform-matrix.yml [hunk 1]
```diff
@@ -206,7 +206,7 @@ jobs:
           - Windows.81.Amd64
           - Windows.10.Amd64
           - Windows.10.Amd64.Core
-          - Windows.10.Nano.Amd64
+          - (Windows.Nano.1803.Amd64.Open)windows.10.amd64.serverrs4.open@mcr.microsoft.com/dotnet-buildtools/prereqs:nanoserver-1803-helix-amd64-05227e1-20190509225944
```
Comment (jkoritzinsky) [inline eng/platform-matrix.yml]: Do we want to be using an open queue here on the internal builds?

Comment (MattGal) [reply eng/platform-matrix.yml]: Definitely not, I missed that.  It will fail trying to send there;  just delete the .open.

**Comment by RussKeldorph** [comment]
@mattgal @jkoritzinsky Thanks for the catch.  I removed both `.Open` and `.open`.  PTAL.

**Pull request merged**